hello world doesn't exactly work out-of-the-box

```
error[E0282]: type annotations needed                                                                                                      
  --> src/main.rs:12:36                                                                                                                    
   |                                                                                                                                       
12 |         MyRule::parse_with(input, |parser, result| {                                                                                  
   |                                    ^^^^^^ consider giving this closure parameter a type                                               
                                                                                                                                           
error: aborting due to previous error                                                                                                      
                                                                                                                                           
For more information about this error, try `rustc --explain E0282`.                                                                        
error: Could not compile `gll-hello-world`.
```

sample code [here](https://github.com/qmx/gll-hello-world/blob/master/src/main.rs#L4-L18)

--- a/README.md
+++ b/README.md
@@ -11,10 +11,12 @@ Easiest way to get started is through `gll-macros`:
 [dependencies]
 gll = "0.0.2"
 gll-macros = "0.0.2"
+proc-macro2 = "0.4.20"
 ```
 ```rust
 extern crate gll;
 extern crate gll_macros;
+extern crate proc_macro2;
 ```
 
 As an example, this is what you might write for a JSON-like syntax,
@@ -39,7 +41,9 @@ You can also use a build script to generate the parser (**TODO**: document).
 
 To parse a string with that grammar:
 ```rust
-let tokens = string.parse().unwrap();
+use proc_macro2::TokenStream;
+
+let tokens: TokenStream = string.parse().unwrap();
 json_like::Value::parse_with(tokens, |parser, result| {
     let value = result.unwrap();
     // ...

